java/modules/basic/src/main/java/io/v6d/modules/basic/arrow/Int64ArrayBuilder.java

```diff
@@ -45,10 +51,10 @@ public ObjectMeta seal(Client client) throws VineyardException {
         meta.setTypename("vineyard::NumericArray<int64>");
         meta.setNBytes(array.getBufferSizeFor(array.getValueCount()));
         meta.setValue("length_", array.getValueCount());
-        meta.setValue("null_count_", 0);
+        meta.setValue("null_count_", array.getNullCount());
         meta.setValue("offset_", 0);
-        meta.addMember("buffer_", buffer.seal(client));
-        meta.addMember("null_bitmap_", BufferBuilder.empty(client));
+        meta.addMember("data_buffer_", dataBufferBuilder.seal(client));
+        meta.addMember("validity_buffer_", validityBufferBuilder.seal(client));
```

Comment:
Don't rename these key names. They MUST be kept consistent with the builders/resolvers in existing C++/Python/Go/Rust SDKs.

Reply:
Done